code/game/machinery/medical_pod/autodoc.dm

```diff
@@ -284,9 +284,7 @@
 	var/mob/living/carbon/human/H = M
 	var/datum/data/record/N = null
```

Comment:
maints will not be happy with single letter vars but I think you didnt put them there so /shrug

Reply:
This pr is huge, imagine if I attempted to change all the single letter vars?